app/scripts/modules/core/src/search/searchResult/SearchResultGrid.tsx

```diff
@@ -1,58 +1,61 @@
 import * as React from 'react';
 import { BindAll } from 'lodash-decorators';
 
-import { ISearchResultFormatter } from './searchResultFormatter.registry';
+import { ISearchResultType } from './searchResultsType.registry';
 import { SearchStatus } from './SearchResults';
 
 export interface ISearchResultGridProps {
   searchStatus: SearchStatus;
-  searchResultFormatter: ISearchResultFormatter;
+  searchResultsType: ISearchResultType;
   searchResults: any[];
 }
 
-@BindAll()
-export class SearchResultGrid extends React.Component<ISearchResultGridProps> {
+const NoQuery = () => (
+  <div className="flex-center">
+    <h2>Please enter a search query to get started</h2>
+  </div>
+);
 
-  public componentDidUpdate(): void {
-    const { searchResultFormatter } = this.props;
-    if (searchResultFormatter) {
-      searchResultFormatter.displayRenderer.scrollToTop();
```

Comment:
I should probably add this functionality back in

Reply:
maybe the component never updates

Reply:
With the new components this isn't needed anymore